src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -60,7 +60,7 @@ public void run() {
     };
 
     @Inject
-    ClientAnalytics(
+    public ClientAnalytics(
```

Comment:
Will be nice to make gin optional but agree with Christian on this.  Also needs more work than just making the ctor public because the class is designed to be a singleton so that needs to be enforced in someway as well.